browser/plugins/test/strip-query-string.test.js

```diff
@@ -0,0 +1,57 @@
+// magical jasmine globals
+const { describe, it, expect } = global
+
+const plugin = require('../strip-query-string')
+
+const Client = require('../../../base/client')
+const VALID_NOTIFIER = { name: 't', version: '0', url: 'http://' }
+
+describe('plugin: strip query string', () => {
+  it('should strip querystrings from urls', () => {
+    expect(
+      plugin._strip('https://bath.tub.xyz/assets/js/app.js?id=49c3ad475f69072745e3')
```

Comment:
Will it strip query strings with multiple items?

```js
plugin._strip('https://bath.tub.xyz/assets/js/app.js?id=49c3ad475f69072745e3&name=bob')
```

Reply:
It will but there isn't a test. I'll add one.